src/commcare_cloud/commands/deploy/slack.py

```diff
@@ -56,6 +59,8 @@ def send_deploy_end_message(self, context, is_success):
         else:
             message = "*Deploy Failed* :x:"
         blocks = self._get_message_blocks(message, context)
+        emoji = "white_check_mark" if is_success else "x"
```

Comment:
Kind of a nit but not really (I don't think it matters that much), the emoji "code names" are easy to miss sprinkled around in message strings. Since they're Slack-specific, it would be nice to store them in a more prominent place and insert them with string formatting.

Reply:
Well, string formatting isn't applicable to the specific line I commented on here. I was referring to how they're hard-coded throughout.

Reply:
Maybe use named constants?